app/src/main/res/layout/activity_edit_profile.xml

```diff
@@ -70,13 +70,13 @@
                     android:id="@+id/EditProfileActivity_Rank"
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
+                    android:editable="false"
                     android:layout_gravity="center_vertical"
                     android:drawableStart="@drawable/ic_baseline_upgrade_24"
                     android:drawablePadding="10dp"
                     android:paddingStart="20dp"
                     android:paddingEnd="0dp"
```

Comment:
Is it really useful to define the padding to 0? Is it not by default?

Reply:
Yes, it's needed otherwise, the XML gave warning about symmetry :)